internal/cli/kraft/cloud/instance/stop/stop.go

```diff
@@ -21,11 +22,11 @@ import (
 )
 
 type StopOptions struct {
-	TimeoutMS int64  `local:"true" long:"timeout-ms" short:"w" usage:"Timeout for the instance to stop"`
-	Output    string `long:"output" short:"o" usage:"Set output format. Options: table,yaml,json,list" default:"table"`
-	All       bool   `long:"all" usage:"Stop all instances"`
-	Metro     string `noattribute:"true"`
-	Token     string `noattribute:"true"`
+	DrainTimeout time.Duration `local:"true" long:"drain-timeout" short:"d" usage:"Timeout for the instance to stop (ms/s/m/h)"`
```

Comment:
"d" is not very good at conveying the idea of a "timeout" IMO.
I would understand "w" for "wait", or if "d" was a bool for drain yes/no.

Is it relevant to change the flag in this PR?

Reply:
I could do 2 PRs, but @felipehuici asked for it be renamed to `drain-timeout`, and people were just using the word "drain" and variations everywhere so I just thought to switch to `-d`

Reply:
here: https://discord.com/channels/879723098881028167/1088815657673572445/1213061589264113664

Reply:
we could also use `-t` for timeout :seal:

Reply:
I take it `-d` is fine :D

Reply:
Will merge now